storey/utils.py

```diff
@@ -304,3 +307,21 @@ def find_filters(partitions_time_attributes, start, end, filters, filter_column)
     # for start=1.2.2018 08:53:15, end=5.2.2018 16:24:31, this method will append to filters
     # [(year=2018, month=2,day<=5, filter_column<5.2.2018 16:24:31)]
     _find_filter_helper(partitions_time_attributes, end, "<", "<=", first_uncommon, filters, filter_column)
+
+
+def drop_reserved_columns(df):
+    cols_to_drop = []
+    for col in df.columns:
+        if col.startswith('igzpart_'):
+            cols_to_drop.append(col)
+    df.drop(labels=cols_to_drop, axis=1, inplace=True, errors='ignore')
+
+
+def asyncify(fn):
+    """Run a synchronous function asynchronously."""
+    async def inner_fn(*args, **kwargs):
+        loop = asyncio.get_event_loop()
+        with concurrent.futures.ThreadPoolExecutor() as pool:
```

Comment:
Hm, is this creating a new executor every time? In existing storey code, we pass `None` for the executor, to indicate the default instance should be used.

Reply:
Need to dig in. Not my code

Reply:
yes, it's using multiple threads.
I believe the motivation is to workaround the Redis client not being async.
i moved this function into the Redis driver, so as to reuse the same pool for all requests.